examples/slack-kit-example/app/actions/fetch-user-id.ts

```diff
@@ -0,0 +1,63 @@
+"use server";
+
+import { Knock } from "@knocklabs/node";
+
+import { getAppDetails } from "../lib/app-details";
+
+const knockApiKey = process.env.KNOCK_API_KEY;
+const knock = new Knock(knockApiKey);
+const appDetails = getAppDetails();
+
+export async function fetchUserId(email: string): Promise<string> {
+  const channelData = await getKnockChannelData();
+  const response = await fetch(
+    `https://slack.com/api/users.lookupByEmail?email=${email}`,
+    {
+      headers: {
+        Authorization: `Bearer ${channelData.data.token.access_token}`,
+      },
+    },
+  );
+  const data = await response.json();
+  console.log("data", data);
```

Comment:
Do you want to keep these console logs in? If so, maybe worth formatting them a bit more, like by making the text more detailed (e.g. `console.log("[Knock] User data:", data);`

Reply:
Good catch, removed the ones that weren't error related